Error handling, parsed message instead of throwing (#21)
th2-net/th2-codec: README.md

## Changelog

> ### v4.1.2

Reviewer: Please update version in the head of document

Reviewer: ```suggestion
### v4.2.0
```

#### Feature:

> * In case of decoding error instead of skipping group replace raw messages of empty or target protocol with `th2-codec-error` message in them```

Reviewer 2: ```suggestion
* In case of decoding error, instead of skipping the group, replace raw messages of empty or target protocol with `th2-codec-error` message in them
```

> * Parsed message with `th2-codec-error` type for error result group. Each raw message with current protocol will be converted to error in result, other stay same

Reviewer 3: ```suggestion
* In case of decoding error instead of skipping group replace raw messages of empty or target protocol with `th2-codec-error` message in them```
